tests/scripts/remote_monitoring_tests/helpers/utils.py

```diff
@@ -381,49 +392,55 @@ def validate_container(update_results_container, update_results_json, list_reco_
-                        # Validate timestamps [deprecated as monitoring end time is moved to higher level]
-                        # assert cost_obj[term]["monitoring_end_time"] == interval_end_time, \
-                        #    f"monitoring end time {cost_obj[term]['monitoring_end_time']} did not match end timestamp {interval_end_time}"
-
-                        monitoring_start_time = term_based_start_time(interval_end_time, term)
-                        assert terms_obj[term]["monitoring_start_time"] == monitoring_start_time, \
-                            f"actual = {terms_obj[term]['monitoring_start_time']} expected = {monitoring_start_time}"
-
-                        # Validate duration in hrs
-                        if expected_duration_in_hours == None:
-                            diff = time_diff_in_hours(interval_start_time, interval_end_time)
-                            print(f"difference in hours = {diff}")
-                            duration_in_hours += diff
-                            print(f"duration in hours = {duration_in_hours}")
-
-                            if term == "short_term" and duration_in_hours > SHORT_TERM_DURATION_IN_HRS_MAX:
-                                duration_in_hours = SHORT_TERM_DURATION_IN_HRS_MAX
-                            elif term == "medium_term" and duration_in_hours > MEDIUM_TERM_DURATION_IN_HRS_MAX:
-                                duration_in_hours = MEDIUM_TERM_DURATION_IN_HRS_MAX
-                            elif term == "long_term" and duration_in_hours > LONG_TERM_DURATION_IN_HRS_MAX:
-                                duration_in_hours = LONG_TERM_DURATION_IN_HRS_MAX
-
-                        print(
-                            f"Actual = {terms_obj[term]['duration_in_hours']} expected = {duration_in_hours}")
-                        assert terms_obj[term]["duration_in_hours"] == duration_in_hours, \
-                            f"Duration in hours did not match! Actual = {terms_obj[term]['duration_in_hours']} expected = {duration_in_hours}"
-
-                        # Get engine objects
-                        engines_list = ["cost", "performance"]
-
-                        # Extract recommendation engine objects
-                        recommendation_engines_object = None
-                        if "recommendation_engines" in terms_obj[term]:
-                            recommendation_engines_object = terms_obj[term]["recommendation_engines"]
-                        if None != recommendation_engines_object:
-                            for engine_entry in engines_list:
-                                if engine_entry in terms_obj[term]["recommendation_engines"]:
-                                    engine_obj = terms_obj[term]["recommendation_engines"][engine_entry]
-                                    validate_config(engine_obj["config"])
+                current_config = list_reco_container["recommendations"]["data"][interval_end_time]["current"]
+
```

Comment:
@khansaad - For loop to verify all the terms is required as the long term test would have all 3 terms and medium term will have short term as well and it needs to be validated